symfony/framework-bundle/4.3/config/bootstrap.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
This file seems to be a duplicate of https://github.com/symfony/recipes/blob/2eafd3c75425318e461b95c1a1ed0c0b3b29f273/symfony/framework-bundle/4.2/config/bootstrap.php

Reply:
we can use a symlink here instead (as we do already)